.gitignore

```diff
@@ -8,3 +8,7 @@ lib/
 # Gatsby directories
 .cache/
 public/
+
+# Mac files
+.DS_Store
```

Comment:
i always recommend configuring developer specific ignores in a [global file](https://github.com/travi/dotfiles/blob/63c76bafe887a74fc0b18154f68e085ec7ebe5ce/git/.gitignore#L8) instead of at the per-project level